streams/src/test/java/org/apache/kafka/streams/integration/ResetIntegrationTest.java

```diff
@@ -42,31 +38,34 @@
 
     @ClassRule
     public static final EmbeddedKafkaCluster CLUSTER;
-    private static final long CLEANUP_CONSUMER_TIMEOUT = 2000L;
-    private static final String APP_ID = "Integration-test";
-    private static final String NON_EXISTING_TOPIC = "nonExistingTopic";
-    private static int testNo = 1;
+
+    private static final String TEST_ID = "reset-integration-test";
 
     static {
-        final Properties props = new Properties();
+        final Properties brokerProps = new Properties();
         // we double the value passed to `time.sleep` in each iteration in one of the map functions, so we disable
         // expiration of connections by the brokers to avoid errors when `AdminClient` sends requests after potentially
         // very long sleep times
-        props.put(KafkaConfig$.MODULE$.ConnectionsMaxIdleMsProp(), -1L);
-        CLUSTER = new EmbeddedKafkaCluster(NUM_BROKERS, props);
-        cluster = CLUSTER;
+        brokerProps.put(KafkaConfig$.MODULE$.ConnectionsMaxIdleMsProp(), -1L);
+        CLUSTER = new EmbeddedKafkaCluster(1, brokerProps);
     }
 
     @AfterClass
-    public static void globalCleanup() {
-        afterClassGlobalCleanup();
+    public static void afterClass() {
```

Comment:
Can we move the `@AfterClass` annotation to the super class?

Reply:
Ack.